.gitignore

```diff
@@ -0,0 +1,2 @@
+# We don't want the lock file in Git, because we have a conditional dependency in our Gemfile.
```

Comment:
We actually _do_ want our lockfile in git, to ensure reproducible builds.
Perhaps there is some workaround possible for this in the lockfile (no idea myself tbh).